lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -183,6 +186,33 @@ class FlutterConfiguration {
     defaultValue: 'https://unpkg.com/canvaskit-wasm@$_canvaskitVersion/bin/',
   );
 
+  /// The variant of CanvasKit to download.
+  ///
+  /// Available values are:
+  ///
+  /// * `auto` - the default value. The engine will automatically detect the
+  /// best variant to use based on the browser.
+  ///
+  /// * `full` - the full variant of CanvasKit that can be used in any browser.
+  ///
+  /// * `chromium` - the lite variant of CanvasKit that can be used in
+  /// Chromium-based browsers.
+  CanvasKitVariant get canvasKitVariant {
+    final String? variant = _configuration?.canvasKitVariant;
+    switch (variant) {
+      case 'auto':
+      case null:
```

Comment:
`case null` needs to be handled outside the switch. According to @rakudrama you found a bug in the compiler!

Reply:
Ah, no, this one is already using Dart only types, so it's fine?

Reply:
My first workaround was using `js_util` so I didn't like it much. Now I changed it and removed the `case null:`. I also added a test explicitly to check for this case.

Reply:
```
   info - lib/src/engine/configuration.dart:48:8 - Unused import: 'package:js/js_util.dart'. Try removing the import directive. - unused_import
```

Reply:
Oops!